tests/ballerina-test-integration/src/test/resources/httpService/httpHeaderTest.bal

```diff
@@ -68,19 +61,11 @@ service<http:Service> quoteService bind stockServiceEP {
         path:"/stocks"
     }
     company (endpoint conn, http:Request req) {
-        //string[] headers = req.getHeaders("core");
-        var result = req.getHeaders("core");
-        match result {
-            string[] headers => {
-                json payload = {header1:headers[0] , header2:headers[1]};
-                http:Response res = {};
-                res.setJsonPayload(payload);
-                _ = conn -> respond(res);
-            }
-            any|null => {
-                return;
-            }
-        }
+        string[] headers = req.getHeaders("core");
```

Comment:
you may need to first check, and then get. clientResponse.hasHeaders("person"). Polite programming :)